app/pages/ServiceDiscoveryForm/ServiceDiscoveryForm.jsx

```diff
@@ -22,7 +22,6 @@ const ServiceDiscoveryForm = ({ match }) => {
       eligibilities={eligibilities}
       subcategories={subcategories}
       steps={category.steps}
-      subcategorySubheading={category.subcategorySubheading}
```

Comment:
I think you accidentally reverted some of the changes you made in https://github.com/ShelterTechSF/askdarcel-web/pull/1106.